Client/qtTeamTalk/utilsound.cpp

```diff
@@ -630,6 +669,7 @@ void PlaySoundEvent::playDefaultSoundEvent(const QString& filename)
     static QSoundEffect* effect = nullptr;
     delete effect;
     effect = new QSoundEffect(ttSettings);
+    effect->setAudioDevice(getSelectedOutputAudioDevice());

```

Comment:
What happens if getSelectedOutputAudioDevice() returns QAudioDevice() ?

Reply:
I've rollback to default output device in this case